### PR TITLE
PHP: add use_tls and use_test_ca param to interop tests script

### LINKED ISSUE #3732: Allow --use_tls=BOOLEAN --use_test_ca=BOOLEAN arg syntax for php interop tests.
Looks like php currently only allows --use_tls flag and --use_tls=true is rejected.

This is not urgent but we should eventually change this.

### PR DESCRIPTION
**Pull request by stanley-cheung**
* Fixes #3732 
 * And some general clean up

### CODE REVIEW

**Comment by stanley-cheung** [comment]
Main jenkins build is green. Interop build has the known errors.

**Comment by jtattermusch** [comment]
LGTM 
Can you also update PHPLanguage in run_interop_tests.py to use the --use_tls=true and --use_test_ca=true. It's fine to do that in a followup PR if you prefer to.

**Comment by stanley-cheung** [comment]
I added the `--use_tls=true` flag to `run_interop_tests.py`. And I also added some code to `run_interop_tests.py` to bring the `language.global_env()` environment variables to the `build_interop_image.sh` job. This is to further automate #3818 a bit instead of relying on user to pass that variable from the command line. Jan, can you please take a look at the last 3 commits? Thanks.

**Comment by stanley-cheung** [comment]
The interop tests Jenkins build is looking good for PHP. The 2 auth interop tests failure are addressed in the next PR #3854.

File: tools/run_tests/run_interop_tests.py [hunk 1]
```diff
@@ -220,17 +220,20 @@ def __init__(self):
...
     return (self.client_cmdline_base + _CLOUD_TO_CLOUD_BASE_ARGS +
-            ['--use_tls', '--use_test_ca'])
+            ['--use_tls=true', '--use_test_ca=true'])
 
   def cloud_to_prod_env(self):
     return _SSL_CERT_ENV
 
   def global_env(self):
-    return {}
+    # need to manually copy to each jenkins machine if we run into github
```
Comment (jtattermusch) [inline tools/run_tests/run_interop_tests.py]: Currently, global_env (as well as all other *_env  functions) are used to set the environment variable setting to actually run the interop client and interop server (see python), not for running the build_interop_image.sh script. 

So what you are doing here make usage of global_env pretty inconsistent  and I would be against it.

Comment (stanley-cheung) [inline tools/run_tests/run_interop_tests.py]: Ah, ic. I will address this in a new PR. Thanks for the background.

**Comment by jtattermusch** [comment]
Alright, by objection came after it was merged. Can we remove the changes from last two commits?

**Comment by stanley-cheung** [comment]
Sure. I will do it in a new PR.

**Pull request merged**